### PR TITLE
Update license headers for 2024

### PR DESCRIPTION
**Pull request by sophio-japharidze-sonarsource**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.sonarsource.parent</groupId>
     <artifactId>parent</artifactId>
-    <version>70.0.0.270</version>
+    <version>71.0.0.1292</version>
```
Comment (vojtech-suchy-sonarsource) [inline pom.xml:7]: Double checking - was this change intended?

**Review by vojtech-suchy-sonarsource — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (sophio-japharidze-sonarsource) [reply pom.xml:7]: Yes, because something related to licenses is coming from there. without this change build will fail
